hawkular-alerts-rest/hawkular-alerts-rest-api/src/main/java/org/hawkular/alerts/rest/ResponseUtil.java

```diff
@@ -84,6 +86,23 @@ public static Response badRequest(String message) {
                 .entity(new ApiError(message)).type(APPLICATION_JSON_TYPE).build();
     }
 
+    public static void checkForUnknownQueryParams(final UriInfo uri, final Set<String> expected) {
```

Comment:
This approach looks good to me.
I have the naive hope that perhaps RESTEasy should offer some kind of helper on this in the forms of flags or built-in checker but if not I would attack this issue in the same way.